Lib/test/test_genericalias.py

```diff
@@ -13,6 +13,7 @@
 from dataclasses import Field
 from functools import partial, partialmethod, cached_property
 from mailbox import Mailbox, _PartialFile
+
```

Comment:
Please don't edit whitespace in areas you're not otherwise touching.

```suggestion
```

Reply:
Oops, I blame my IDE for auto-formatting that in 🤦 , I'll be more careful in the future.